jupyter_book/07-preprocessing-steps/preprocessing-steps.md

```diff
@@ -0,0 +1,17 @@
+# Preprocessing steps
+
+Before inputting the data into the satay pipeline, it is necessary to preprocess the data that comes from the sequencing company. 
+
+The pipeline understands single end sequencing data, in which the data from every digestion is mixed together. You dont need to analyze them separately. 
```

Comment:
Not clear here what 'understands' means exactly. It should be mentioned that if the pipeline in its current state does not treat the two digestions differently and will use the sequence that is downstream of the digestion sites (which is a sequence from the plasmid) for the mapping of the sequence to the genome

Reply:
I mean that the pipeline currently is optimized for single end reads from both digestions . I can say instead then: 

_The pipeline is currently optimized for single end reads from both digestions. It is not needed to separate them._

Reply:
It is doubtful to say that it is currently optimized, since it just completely ignores the fact that reads come from different digestions

Reply:
I would actually explicitly state that there is no processing on that part, so it can affect the alignment/results

> The pipeline do**es** (typo) not require single end reads each from separate digestions. So they can be mixed together.

Reply:
> No exactly , that will be the job of the preprocessing steps I am putting here

I would maybe suggest this:

> The pipeline does not process each digestion separately and therefore any pre-processing and trimming of the restriction sites should be done **prior** to running the pipeline.